.github/workflows/build-test-windows.yml

```
@@ -2,11 +2,22 @@
 name: build-and-test-windows
 on:
   push:
-    branches: [ main, master ]
+    branches:
+      - master
+
+    paths-ignore:
+      - '**.md'
+      - '.github/**'
+      - '!.github/workflows/**'
```

This should be updated like the other one, right? 
```suggestion
      - '!.github/workflows/build-*'
```

---

Weird. Thought I updated it too. Let's me fixed it again